docs/common/yum.md

```diff
@@ -172,6 +172,12 @@ You must install and enable these first.
 
 ### Install the OSG Repositories
 
+!!! danger "Before considering an upgrade to OSG 3.6&hellip;"
```

Comment:
```suggestion
!!! danger "Before considering an upgrade to OSG 3.6"
```
Unless that's on purpose?

Reply:
That's what we have everywhere else so I figure we'd keep it the same